process/charter.html

```diff
@@ -294,7 +289,7 @@ <h3>5.1 Organizing the Call for Review</h3>
     <ol>
       <li>A URI to the proposed charter (Note: avoid using github.io links); this charter is public during the AC review.</li>
```

Comment:
We don't want to send github.io links for AC review.

```suggestion
      <li>A URI to the proposed charter (don't use github.io links); this charter is public during the AC review.</li>
```